server/routes/activities/deallocate-from-activity/journey.ts

```diff
@@ -11,4 +11,5 @@ export type DeallocateFromActivityJourney = {
   }>
   deallocationDate?: SimpleDate
   deallocationReason?: string
+  earliestAllocationStartDate?: string
 }
```

Comment:
Do these values get cleared from the session journey when a) it starts, and b) when it is actioned?

Reply:
a) The values are set within the DEALLOCATE part of the allocation dashboard which is the start of the deallocate journey, so no need to clear.
b) Yes, the session object is cleared in the confirmation handler of the deallocation journey (line 7)